src/main/java/com/crud/tasks/service/MailCreatorService.java

```diff
@@ -16,12 +17,18 @@
     @Autowired
     AdminConfig adminConfig;
 
+    @Autowired
+    CompanyData companyData;
+
     public String buildTrelloCardEmail(String message) {
         Context context = new Context();
         context.setVariable("message", message);
         context.setVariable("task_url", "http://localhost");
         context.setVariable("button", "Visit website");
         context.setVariable("admin_name", adminConfig.getAdminName());
+        context.setVariable("company_name", companyData.getCompanyName());
```

Comment:
can you check if it will be possible to set here just one object: companyData and in template, use for example ${companyData.Name}